tests/zeek/double-type-registration.zeek

```diff
@@ -0,0 +1,32 @@
+# @TEST-EXEC: spicyz -L . -o a.hlto a.spicy
+# @TEST-EXEC: spicyz -L . -o b.hlto b.spicy
```

Comment:
Let's drop the `-L .` here like we do below. It should make no difference, but made me wonder whether it does.

Reply:
I needed it, otherwise it didn't compile.